samples/boards/nrf/ieee802154/802154_rpmsg/README.rst

```diff
@@ -0,0 +1,32 @@
+.. _nrf-ieee802154-rpmsg-sample:
+
+nRF IEEE 802.15.4: Serialization RPMsg
+######################################
+
+Overview
+********
+
+This sample exposes IEEE 802.15.4 radio driver support
+to another device or CPU using RPMsg transport which is
+a part of `OpenAMP <https://github.com/OpenAMP/open-amp/>`__.
+
+Requirements
+************
+
+* A board with :ref:`ipm_api` driver and nRF IEEE 802.15.4 radio driver support
```

Comment:
This should clearly state that this will only work on Nordic boards

Reply:
I've made it clear by replacing conditions with the only chip meeting all conditions (nRF53)